### DISCUSSION TITLE
Welcome to HID Discussions!

### DISCUSSION THREAD

**Discussion by NicoHood** [NicoHood/HID#250] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by ghost** [comment]
How to download HID project for Arduino UNO please explain step by step.

**Reply by NicoHood** [reply]
https://github.com/NicoHood/HID/wiki/Installation

**Comment by John-Silver-i2p** [comment]
Привет, объясните пожалуйста как сделать так чтобы при выполнение кода в консоли писалась команда start а не stat !?
вот код #include &amp;lt;Keyboard.h&amp;gt;
void setup() {
  Keyboard.begin();
  delay(3000);//Delay
  Keyboard.press(KEY_LEFT_GUI);//Кнопка win
  delay(500); 
  Keyboard.press('r');//Win+R
  delay(500); 
  Keyboard.release(KEY_LEFT_GUI);
  Keyboard.println("cmd"); // вызываем командную строку
  Keyboard.press(KEY_RETURN);
  delay(500);
  Keyboard.release(KEY_RETURN);
  delay(500); 
  Keyboard.println("start file");
  Keyboard.press(KEY_RETURN);
  delay(500);
  Keyboard.release(KEY_RETURN);
  Keyboard.end();
}
void loop()
{
}

**Comment by andreo32** [comment]
Hello.
I want to create HID keyboard using Arduino Pro Micro. Is it possible to use ports 0 and 1 (Rx and Tx) as usual digital port (for example to drive two Led diodes) for HID keyboard?

**Reply by NicoHood** [reply]
That is a general arduino question and not related to this library. But yes, you should be able to, but I think you need to disable the serial interface maybe (the ports might be preconfigured, I am not 100% sure)

**Comment by ImranMazlan** [comment]
hello, 
I want to use the HID library but most the examples from the library that I had been used always generated same error even thought that I already download the hoodloader2 and my IDE are the latest. This is the error:-

![Screenshot 2023-01-05 123046](https://user-images.githubusercontent.com/121958059/210702514-a61e3abc-fe8d-45d9-9ac3-34277bc924db.png)

**Reply by nirajandata** [reply]
bro, do ctrl+shift+i and open library manager and then search for hid project and after that install that library.

**Reply by MichaelSchichta** [reply]
Make sure to do
#include "Arduino.h" before
#include <HID-Project.h>
#include <Keypad.h>

Had the same issue, this solved it for me

**Comment by ImranMazlan** [comment]
Thank you for replying
